src/plone/restapi/serializer/relationfield.py

```diff
@@ -17,7 +18,12 @@
 @adapter(IRelationValue)
 @implementer(IJsonCompatible)
 def relationvalue_converter(value):
-    if value.to_object:
+    has_permission = api.user.has_permission(
```

Comment:
@plone/restapi-team In a comment above @erral wrote `plone.api` usage is allowed in this package in tests only and asked if this is right. May one you confirm this?

Reply:
We agreed that we can use plone.api in tests. Though, if I am not mistaken we also decided lately that plone.restapi can use plone.api everywhere, correct? Sorry for the late reply here. Just want to get this straight.